cypress/e2e/fee-fine/input-incorrect-value-to-bill-actual-cost.cy.js

```diff
@@ -0,0 +1,191 @@
+import UsersSearchResultsPane from '../../support/fragments/users/usersSearchResultsPane';
+import BillActualCostModal from '../../support/fragments/users/billActualCostModal';
+import InventoryInstances from '../../support/fragments/inventory/inventoryInstances';
+import LostItemFeePolicy from '../../support/fragments/circulation/lost-item-fee-policy';
+import CirculationRules from '../../support/fragments/circulation/circulation-rules';
+import UsersSearchPane from '../../support/fragments/users/usersSearchPane';
+import UsersOwners from '../../support/fragments/settings/users/usersOwners';
+import LoanDetails from '../../support/fragments/users/userDefaultObjects/loanDetails';
+import UsersCard from '../../support/fragments/users/usersCard';
+import UserLoans from '../../support/fragments/users/loans/userLoans';
+import UserEdit from '../../support/fragments/users/userEdit';
+import Checkout from '../../support/fragments/checkout/checkout';
+import TopMenu from '../../support/fragments/topMenu';
+import Users from '../../support/fragments/users/users';
+
+describe('Lost items requiring actual cost', () => {
+  const testData = {
+    folioInstances: InventoryInstances.generateFolioInstances(),
+    servicePoint: ServicePoints.getDefaultServicePoint(),
+  };
+  const declareLostComments = getTestEntityValue('Some additional information');
+  const ownerData = UsersOwners.getDefaultNewOwner();
+  const feePolicy = {
+    name: getTestEntityValue('1-minute-test'),
+    chargeAmountItem: {
+      chargeType: 'actualCost',
+      amount: 0.0,
+    },
+    lostItemProcessingFee: 25.0,
+    chargeAmountItemPatron: true,
+    chargeAmountItemSystem: true,
+    itemAgedLostOverdue: {
+      duration: 1,
+      intervalId: 'Minutes',
+    },
+    lostItemChargeFeeFine: {
+      duration: 6,
+      intervalId: 'Weeks',
+    },
+    feesFinesShallRefunded: {
+      duration: 6,
+      intervalId: 'Months',
+    },
+    patronBilledAfterAgedLost: {
+      duration: 1,
+      intervalId: 'Minutes',
+    },
+    returnedLostItemProcessingFee: false,
+    replacedLostItemProcessingFee: false,
+    replacementProcessingFee: 0.0,
+    replacementAllowed: false,
+    lostItemReturned: 'Charge',
+    id: uuid(),
+  };
+  let itemData;
+
+  before('Create test data', () => {
+    cy.getAdminToken().then(() => {
+      ServicePoints.createViaApi(testData.servicePoint);
+      testData.defaultLocation = Locations.getDefaultLocation({
+        servicePointId: testData.servicePoint.id,
+      }).location;
+      Locations.createViaApi(testData.defaultLocation).then((location) => {
+        InventoryInstances.createFolioInstancesViaApi({
+          folioInstances: testData.folioInstances,
+          location,
+        });
+      });
+
+      cy.createLoanType({
+        name: getTestEntityValue('feeFine'),
+      })
+        .then((loanType) => {
+          testData.loanTypeId = loanType.id;
+        })
```

Comment:
please update your item loan type otherwise the circulation rules will not work.
`cy.getItems({ limit: 1, expandAll: true, query: `"barcode"=="${item.barcode}"` }).then(
          (res) => {
            res.permanentLoanType = { id: testData.loanTypeId };
            cy.updateItemViaApi(res);
          },
        );`